codesign/bin/codesign.dart

```diff
@@ -34,13 +32,21 @@ const String kGcsUploadPathOption = 'gcs-upload-path';
 /// For [kGcsDownloadPathOption] and [kGcsUploadPathOption], they are required parameter to specify the google cloud bucket paths.
 /// [kGcsDownloadPathOption] is the google cloud bucket prefix to download the remote artifacts,
 /// [kGcsUploadPathOption] is the cloud bucket prefix to upload codesigned artifact to.
-/// For example, supply '--gcs-download-path=gs://flutter_infra_release/ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip',
+/// For example, supply '--GCS-DOWNLOAD-PATH=gs://flutter_infra_release/ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip',
 /// and code sign app will download the artifact at 'flutter_infra_release/ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip' on google cloud storage
 ///
+/// For [kCredentialsPathOption], this is the file path of the password file in file system. The password file stores sensitive passwords.
+/// sensitive passwords include <CODESIGN-APPSTORE-ID>, <CODESIGN-TEAM-ID>, and <APP-SPECIFIC-PASSWORD>.
+/// For example, if a user supplies --PASSWORD-FILE-PATH=/tmp/passwords.txt, then we would be expecting a password file located at /tmp/passwords.txt.
+/// The password file should provide the password value for each of the password name, deliminated by a single colon. The content of a password file would look similar to:
+/// CODESIGN-APPSTORE-ID:123
+/// CODESIGN-TEAM-ID:456
+/// APP-SPECIFIC-PASSWORD:789
+///
 /// Usage:
 /// ```shell
-/// dart run bin/codesign.dart --[no-]dryrun --gcs-download-path=gs://flutter_infra_release/flutter/<commit>/android-arm-profile/artifacts.zip
-/// --gcs-upload-path=gs://flutter_infra_release/flutter/<commit>/android-arm-profile/artifacts.zip
+/// dart run bin/codesign.dart --[no-]dryrun --GCS-DOWNLOAD-PATH=gs://flutter_infra_release/flutter/<commit>/android-arm-profile/artifacts.zip
```

Comment:
lower case for flags?

Reply:
Sure, i lowered case for file paths, and kept upper case for cert name and user name.